pipelines/controller/src/main/java/com/google/fhir/analytics/ApiController.java

```diff
@@ -166,4 +178,60 @@ public ResponseEntity<String> testView(
     }
     return new ResponseEntity<>(response, status);
   }
+
+  @GetMapping("/next")
+  public ScheduleDto getNextScheduled() {
+    ScheduleDto schedule = new ScheduleDto();
+    LocalDateTime nextRun = pipelineManager.getNextIncrementalTime();
+    if (nextRun == null) {
+      schedule.setNextRun("NOT SCHEDULED");
+    } else {
+      schedule.setNextRun(nextRun.toString());
+    }
+    return schedule;
+  }
+
+  @GetMapping("/dwh")
+  public DwhDto getDwh() {
+    DwhDto dwhDto = new DwhDto();
+    String dwh = pipelineManager.getCurrentDwhRoot();
+    dwhDto.setDwhPrefix(dataProperties.getDwhRootPrefix());
+    dwhDto.setDwhPath(dwh == null || dwh.isEmpty() ? "" : dwh);
+    dwhDto.setDwhSnapshots(dwhFilesManager.listDwhSnapshots());
+    return dwhDto;
+  }
+
+  @DeleteMapping("/dwh")
+  @ResponseStatus(HttpStatus.NO_CONTENT)
+  public void deleteSnapshot(@RequestParam String snapshotId) {
+    dwhFilesManager.deleteDwhSnapshotFiles(snapshotId);
+  }
+
+  @GetMapping("/config")
+  public Map<String, String> getConfigs() {
+    return getConfigMap(null);
+  }
+
+  @GetMapping("/config/{name}")
+  public Map<String, String> getConfigs(@PathVariable String name) {
```

Comment:
Should we return a single string in this case or do you want the name of the config param included in the response too?

Reply:
This was mainly for consistency with the getAll endpoint, but you have raised a good point.

I don't feel strongly for either here, let me know if you prefer single string approach only.

Reply:
We can go ahead with the JSON option for now.